src/components/TooltipButton/TooltipButton.tsx

```diff
@@ -22,6 +23,7 @@ export const TooltipButton = (props: TooltipButtonProps) => {
       className={classNames("tooltip-button", `tooltip-button--${props.direction ?? "right"}`, {"tooltip-button--active": props.active}, props.className)}
       onClick={() => props.onClick()}
       aria-label={props.label}
+      data-testid={props.dataTestid}
```

Comment:
Using data test ids should be avoided if possible since it doesn't resemble how our application is used

Reply:
Removed it